### ISSUE TITLE
[Bug]: Extreme globalShortcut flakiness on Kubuntu 24.04 Wayland

### ISSUE THREAD

**Issue by Nantris** [electron/electron#42573]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.

### Electron Version

31.0.1

### What operating system are you using?

Ubuntu

### Operating System Version

Kubuntu 24.04

### What arch are you using?

x64

### Last Known Working Electron version

30.1.x

### Expected Behavior

globalShortcut works from all applications

### Actual Behavior

globalShortcut is ignored in some application (eg Sublime Text) despite not having any interfering local shortcut in that application.

### Additional Information

It works fine if the focused application is Chrome, Android Studio, or the Android Emulator

It fails if I focus Sublime Text, Firefox, Dolphin, or no applications.

Specifically testing using `Ctrl+Shift+Z` and `Ctrl+Shift+X`

This seems to have been reported years ago and simply rejected as a `wontfix`? #15863 That would make all our years of development a waste if that's really not going to be fixed.

**Comment by Nantris** [comment]
@VerteDinde I cannot test this Electron Fiddle [since Electron Fiddle is broken in Kubuntu 24.04](https://github.com/electron/electron/issues/42510#issuecomment-2181687439), but this problem is exceptionally straightforward to demonstrate and this code should work: https://gist.github.com/07f00c727fca205776f8abae9aec3c40

**Comment by anereganaELH** [comment]
I have the same trouble with Wayland protocol on my ubuntu 22.04. Changing the display protocol to x11 by disabling Wayland works for me for now, but I do think this problem should be solved.

**Comment by Nantris** [comment]
@anereganaELH the only issue with a potential way forward mentioned is #38288 which I'd recommend you upvote, and comment on if you see fit.

**Comment by Nantris** [comment]
Absolutely not stale...

**Comment by Nantris** [comment]
Definitely not stale.

**Comment by Nantris** [comment]
This should be resolved now.

**Comment by Nantris** [comment]
This is not resolved.

**Comment by anereganaELH** [comment]
Has it been resolved?

**Comment by Nantris** [comment]
@anereganaELH no, this is completely unresolved in `37.2.5` and may not be resolvable without special attention from the Electron team. I hope they'll work to address this because it's a major breach of the advertised feature set, cutting whole applications down at the knees.

See #45171.

**Comment by Nantris** [comment]
Not stale.

**Comment by Nantris** [comment]
I understand the Electron-side may exist but may require OS configuration. And I understand Linux is fragmented and docs can't cover every possible configuration. 

Unfortunately, without a single example it remains unclear whether this is fixed and what action users need to take outside of Electron. 

Official guidance from the Electron team would be very welcome.

**Comment by Nantris** [comment]
Not really stale because it remains unclear how to configure global shortcuts with Wayland on the OS side (even one distro with an example would go a very long way towards making this closasble)
